### PR TITLE
Notif

### PR DESCRIPTION
**Pull request by RePeet13**
Updated application to use notifications:
-Toast when topics are loaded/randomized
-Notification on Win or Loss (currently implemented as button)

### CODE REVIEW

**Comment by RePeet13** [comment]
@dgreenhalgh or @abettadapur please take a look over the changes and merge the request if you think its good to go. From there we can merge into master and create a tag or w/e is needed.

File: TopicBingo/res/layout/activity_play.xml [hunk 1]
```diff
@@ -1,48 +1,84 @@
...
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="2"
+                android:onClick="RandomizeTopics"
+                android:text="@string/randomize_button" />
+
+        </LinearLayout>
+    </LinearLayout>
+
+</ScrollView>
```
Comment (dgreenhalgh) [inline TopicBingo/res/layout/activity_play.xml]: Newline

File: TopicBingo/res/values/strings.xml [hunk 2]
```diff
@@ -35,11 +35,37 @@
...
+        <item>Racing Greyhounds</item>
+        <item>Worst Way to Die</item>
+        <item>Your Idol and Why</item>
+        <item>Your Favorite Collection</item>
+        <item>Your Opinion on Javadoc</item>
+        <item>Your First Vehicle</item>
+        <item>Their Ex-(boy/girl)friends</item>
+        <item>Cheddar Cheese</item>
+        <item>Wok</item>
+        <item>Montezuma\'s Revenge</item>
```
Comment (dgreenhalgh) [inline TopicBingo/res/values/strings.xml:60]: I'm liking all of these

File: TopicBingo/res/values/strings.xml [hunk 3]
```diff
@@ -35,11 +35,37 @@
...
 
 
 
     </string-array>
 
-    <string name="max">5/5</string>
+    <string name="max">0/5</string>
```
Comment (dgreenhalgh) [inline TopicBingo/res/values/strings.xml]: Is `max` still a good name for this variable?

File: TopicBingo/src/org/gitmad/topicbingo/TopicBingoApplication.java [hunk 4]
```diff
@@ -1,26 +1,132 @@
 package org.gitmad.topicbingo;
```
Comment (dgreenhalgh) [inline TopicBingo/src/org/gitmad/topicbingo/TopicBingoApplication.java:1]: There are several strings in this file that could be added to strings.xml for good practice

File: TopicBingo/src/org/gitmad/topicbingo/view/PlayActivity.java [hunk 5]
```diff
@@ -48,4 +54,21 @@ public boolean onCreateOptionsMenu(Menu menu) {
...
+				.sendNotification(TopicBingoApplication.notificationType.NOTIFY_WIN);
+	}
+
+	public void notifyLoss(View v) {
+		((TopicBingoApplication) getApplication())
+				.sendNotification(TopicBingoApplication.notificationType.NOTIFY_LOSE);
+	}
+
+	public void RandomizeTopics(View v) {
+		// TODO randomize topics here
```
Comment (dgreenhalgh) [inline TopicBingo/src/org/gitmad/topicbingo/view/PlayActivity.java]: Is this TODO still applicable?

File: TopicBingo/res/values/strings.xml (hunk 3 above)
Comment (dgreenhalgh) [inline TopicBingo/res/values/strings.xml]: Full disclosure: I was the one who named it `max` in the first place, and it wasn't a great name then

**Comment by dgreenhalgh** [comment]
Just had a few nitpicks, looks good overall to me though. I'm liking the logo

File: TopicBingo/res/values/strings.xml (hunk 3 above)
Comment (RePeet13) [inline TopicBingo/res/values/strings.xml]: changed to "progress" couldn't think of anything better

File: TopicBingo/res/values/strings.xml (hunk 2 above)
Comment (RePeet13) [inline TopicBingo/res/values/strings.xml:60]: Thanks, figured we could stand a few more :P

**Pull request merged**